### PR TITLE
v0.2.1

### PR DESCRIPTION
**Pull request by nolanmar511**
Release notes at: https://github.com/GoogleCloudPlatform/cloud-profiler-nodejs/releases/edit/untagged-a6172fd66f9d89f17903

### CODE REVIEW

**Comment by aalexand** [comment]
@nolanmar511 There is an npmaudit failure?

**Comment by nolanmar511** [comment]
npm audit had a flake, causing tests to fail. 
I re-ran and tests passed.

I've also updated this branch (and the release notes) to include a commit added yesterday.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/profiler",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "Adds support for Google Cloud Profiler to node.js applications",
```
Comment (aalexand) [inline package.json]: "Google Cloud Profiler" -> "Stackdriver Profiler"

Comment (nolanmar511) [reply package.json]: Done.

**Pull request merged**